map_model/src/make/buildings.rs

```diff
@@ -150,7 +162,11 @@ fn classify_bldg(
             "public",
         ],
     ) {
-        return BuildingType::Commercial;
+        // 1 person per 10 square meters
+        // TODO: Hone in this parameter. Space per person varies with (among other things):
+        //  - building type. e.g. office vs. warehouse
+        //  - regional/cultural norms
```

Comment:
This is a crude "first pass". We could surely do better, but inevitably there will be non-trivial assumptions encoded in this kind of logic. 

It might make sense to eventually have these assumption ultimately be part of the travel demand model (or some other model).

Reply:
> It might make sense to eventually have these assumption encoded into the travel demand model (or some other model).

It depends if we ever plan to bring in more authoritative landuse or census data. If we do, it'll happen at import-time, so it's a property of the maps. Things like the UI may also want to use it, not just the demand model. For the meantime, baking it into the map from these assumptions seems fine to me.

Reply:
Sounds good.